tests/Microsoft.Identity.Test.Unit/ResourceHelper.cs

```diff
@@ -18,12 +18,7 @@ public class ResourceHelper
         /// </remarks>
         public static string GetTestResourceRelativePath(string resourceName)
         {
-
-#if DESKTOP
```

Comment:
Is this check no longer needed?

Reply:
Looks like the updated the mstest dependencies "fixed" this inconsistency between .NET and .NET Fwk